pom.xml

```diff
@@ -149,61 +159,152 @@
             <format>html</format>
             <format>xml</format>
           </formats>
+          <check/>
         </configuration>
       </plugin>
     </plugins>
   </build>
 
+  <distributionManagement>
+    <repository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/releases</url>
+    </repository>
+  </distributionManagement>
+
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </repository>
+    <repository>
+      <id>repo.jenkins-ci.org-releases</id>
+      <url>https://repo.jenkins-ci.org/releases/</url>
     </repository>
+
   </repositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org-pub</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </pluginRepository>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/releases/</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.222.x</artifactId>
+        <version>26</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
 
   <dependencies>
     <!-- com.google.apis -->
-    <!-- https://developers.google.com/api-client-library/java/apis/storage/v1 -->
+    <!-- https://mvnrepository.com/artifact/com.google.apis/google-api-services-storage -->
     <dependency>
       <groupId>com.google.apis</groupId>
       <artifactId>google-api-services-storage</artifactId>
-      <version>v1-rev12-1.19.0</version>
+      <version>v1-rev171-1.25.0</version>
     </dependency>
 
-    <!-- com.google.guava -->
+    <!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>14.0.1</version>
+      <version>30.1.1-jre</version>
     </dependency>
 
-    <!-- Docker Commons -->
+    <!-- https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-jxr-plugin -->
     <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>docker-commons</artifactId>
-        <version>1.0</version>
+      <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Why would you need to inject Maven plugin codebase into the code?
It is likely to cause a lot of issues, and I do not see usages